_posts/2016-04-25-episerver-developer-meetup-helsinki.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: Episerver developer meetup at Helsinki
+author: Rinorragi
+excerpt: What happened in the first Episerver developer meetup held in Helsinki
+categories: 
+- Episerver
+tags: 
+- Episerver 
+- DOTNET 
+- security
+---
+Episerver had a fine idea to have a meetup for finnish Episerver developers. Meetup consisted of pizza, beer and presentations. I loved to be present on the meetup and I think that the format was a perfect match to have a cozy atmosphere around the Episerver topic. Only thing that could be done better (for the next time) would be to have more time at the end for the networking! Need for more time usually means that the actual happening has been great success! 
```

Comment:
Entä: "Last week EPiServer organized a meetup for Finnish developers in Helsinki"? Luonnollisempaa kieltä etc.